GroupRepresentationsForCAP/gap/Julia.gi

```diff
@@ -0,0 +1,16 @@
+#
+# GroupRepresentationsForCAP: Skeletal category of group representations for CAP
+#
+# Implementations
+#
+
+##
+InstallOtherMethod( RepresentationCategoryObject,
```

Comment:
Could you explain why you need this function (and thus the optional dependency on JuliaInterface)?

Reply:
I think we only want this for convenience so we can write
```
RepresentationCategoryObject( chi, RepG, "\\chi" );
```
in a Jupyter Notebook instead of
```
RepresentationCategoryObject( chi, RepG, g"\\chi" );
```

@mohamed-barakat Is my statement correct or is there some other reason I have missed?

Reply:
> @mohamed-barakat Is my statement correct or is there some other reason I have missed?

Yes, your statement is correct.

Reply:
Then the problem appears to be more general and maybe it would be desirable to have a solution on the Julia side?

If not, we would need to use `JuliaInterface` as an optional package and a method like this one for each instance of a GAP command which uses a string and that we want to call via Julia.

Reply:
I have just pushed a version without the Julia convenience so the discussion does not block the PR. I have no strong opinion on this and will discuss with Mohamed later.

Reply:
Thanks, merged.

Reply:
Like in the other packages I would suggest separate files `Julia.gd/Julia.gi` with the following line in `init.g/read.g`

```
if IsPackageMarkedForLoading( "JuliaInterface", ">= 0.2" ) then
    ReadPackage( "GroupRepresentationsForCAP", "gap/Julia.gi");
fi;
```
```
if IsPackageMarkedForLoading( "JuliaInterface", ">= 0.2" ) then
    ReadPackage( "GroupRepresentationsForCAP", "gap/Julia.gi");
fi;
```

These files hence won't affect the package when used within normal `GAP`, but will be read if the package is loaded using `GAP.jl`.

Reply:
Sorry, I was confused by just looking at the current state of the PR. It seems you have just deleted `Julia.gi` from the PR.

Reply:
For reference: We now handle this as a special case in HomalgProject.jl and will create an issue in GAP.jl to handle it in a more general way in the future.